doc/completion.sh

```diff
@@ -1,4 +1,3 @@
-#!/bin/bash
```

Comment:
This change seems unrelated

Reply:
Yeah, I think I included that by accident while fixing up this for packaging.